### PR TITLE
to_padded_tensor doc v0

### LINKED ISSUE #76846: Document nested_tensor.to_padded_tensor()
### 📚 The doc issue

There's no docs for `to_padded_tensor()` yet. They should be added using `add_docstr_all()` in `torch/_tensor_docs.py`.

```python
import torch

x = torch.nested_tensor((torch.randn(2, 1), torch.randn(2, 2)))
print(x.to_padded_tensor(0.))

# tensor([[[ 0.3007,  0.0000],
#          [-1.0814,  0.0000]],
# 
#         [[-1.5946, -0.9546],
#          [ 2.3071, -0.4019]]])
```

### Suggest a potential alternative/fix

_No response_

cc @svekars @holly1238 @cpuhrsch

### PR DESCRIPTION
**Pull request by YifanShenSZ**
Fixes #76846

### CODE REVIEW

**Review by jbschlosser — COMMENTED** [review]
Thanks for the update! It looks like this didn't actually add the docs to the website, so we need to update an .rst file underneath `docs/source` to reference the function so the docs will end up on the site.

Best place is in `docs/source/nested.rst`. You can use `sparse.rst` as a reference (e.g. `to_dense()`) for how to format the `autosummary` entry for `to_padded_tensor`.

File: docs/source/nested.rst [hunk 1]
```diff
@@ -60,3 +60,10 @@ nested_tensor([
 ])
 >>> nt.unbind()
 [tensor([0., 1., 2.], device='cuda:0'), tensor([3., 4., 5., 6., 7.], device='cuda:0')]
+
```
Comment (jbschlosser) [inline docs/source/nested.rst:63]: To match what's done with sparse, I suggest adding a `Nested tensor methods` section header here, with a description of `The following Tensor methods are related to nested tensors:` before the autosummary

**Review by jbschlosser — COMMENTED** [review]
Looking pretty good! Just a couple minor formatting tweaks and we can ship it :)

**Comment by YifanShenSZ** [comment]
Should be good this time. Let's close it folks!

**Review by jbschlosser — APPROVED** [review]
LGTM!

**Pull request closed without merging**